manila/share/drivers/netapp/dataontap/client/client_cmode.py

```diff
@@ -2120,6 +2120,21 @@ def configure_cifs_options(self):
                 # no raise to be non-blocking
                 LOG.warning(msg, e.message)
 
+        if not security_service['server']:
+            return
+
+        api_args = {
```

Comment:
I think this overrides the previous definition of api_args above under "if not self.features.CIFS_LARGE_MTU".
You need to add a key to the dictionary.